Dockerfile

```diff
@@ -14,8 +14,13 @@ RUN . /opt/ros/mas_stable/setup.sh && \
     rm -rf /var/lib/apt/lists/* && \
     catkin config --init && \
     catkin config --extend /opt/ros/mas_stable && \
-    catkin config --install --install-space /opt/ros/mas_stable && \
-    catkin build && \
+    catkin config --install --install-space /opt/ros/mas_stable
+
+WORKDIR /kinetic/src/mas_domestic_datasets
+RUN . git lfs pull
```

Comment:
Might not be worth it to clone this inside the docker image, unless you want the image to contain the datasets. Otherwise, I would clone this from Travis, then add it as a volume.